lib/hsa/hsaBeamformHFBSum.hpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @brief Sum HFB data over each sample
+ *  - hsaBeamformHFBSum : public hsaCommand
+ */
+
+#ifndef HSA_BEAMFORM_HFB_SUM_H
+#define HSA_BEAMFORM_HFB_SUM_H
+
+#include "hsaCommand.hpp"
+
+/**
+ * @class hsaBeamformHFBSum
+ * @brief hsaCommand to upchannelize and downsample FRB data
+ *
+ * This is an hsaCommand that launches the kernel (sum_hfb) for
+ * summing HFB data across all samples. Final output in float of
+ * 1024 beams x 128 freq.
+ *
+ * @requires_kernel   sum_hfb.hasco
+ *
+ * @par GPU Memory
+ * @gpu_mem  hfb_output  Input data of size input_frame_len
```

Comment:
This is missing the `compressed_lost_samples` memory block.  Also it might be worth adding a note above in the summary about how it excludes samples from the sum where there is 1 or more lots samples.